resources/files/windows/puppet_shell.bat

```diff
@@ -3,4 +3,4 @@
 call "%~dp0environment.bat" %0 %*
 

```

Comment:
This is used to start a new CMD with puppet loaded, should we add `%PUPPET_DIR%\bin\` in PATH to be able to use the executables from there? Not really sure how people are using this.